num/src/bigint.rs

```diff
@@ -16,6 +16,7 @@
 // If not, see <https://opensource.org/licenses/MIT>.
 
 use crate::error::ParseLengthError;
+use std::convert::TryInto;
```

Comment:
we need to use `core::` here to maintain `no_std` compatibility

Reply:
Ok, but I see std in another partes of the code, getters for example

Reply:
I'm trying to figure out reading u64 Code but I'm not sure how to proceed.
If you can give me an indication I would be pleased

Reply:
Getters are part of other crate, which is not `no_std` one. The main indicator is when CI is broken, and here it indeed is.

Reply:
> I'm trying to figure out reading u64 Code but I'm not sure how to proceed.

not sure I understood your question